sklearn/tests/test_metadata_routing.py

```diff
@@ -638,14 +638,14 @@ class Consumer(BaseEstimator):
 @config_context(enable_metadata_routing=True)
 def test_metadata_request_consumes_method():
     """Test that MetadataRequest().consumes() method works as expected."""
-    request = MetadataRouter(owner="test")
+    request = MetadataRequest(owner="test_method")
     assert request.consumes(method="fit", params={"foo"}) == set()
 
-    request = MetadataRequest(owner="test")
+    request = MetadataRequest(owner="test_method")
```

Comment:
I don't understand why the diff here. The owner in the whole test file is mostly `"test"`, and it doesn't really matter what it is. I don't mind it either way, but it's not something I'd go out of my way to change.